### DISCUSSION TITLE
Trying to patch a non-existent item does not indicate that there is no item to update

### DISCUSSION THREAD

**Discussion by BronzW** [directus/directus#12605] [Archive]
### Describe the Bug

If you attempt to use the PATCH method against an item that does not exist in a certain collection, the response status will still be 204 No Content. Showing that the process went well, when it actually didn't.

### To Reproduce

1. Create a node environment
2. Install axios `npm install axios`
`const axios = require('axios')`
`const res = await axios.patch(root+ "/items/:collection/" + id, client, auth)`
`console.log(res.status)`

### Errors Shown

That's the problem

### What version of Directus are you using?

9.7.x

### What version of Node.js are you using?

v16.5.0

### What database are you using?

SQL

### What browser are you using?

Chrome

### How are you deploying Directus?

locally

**Comment by rijkvanzanten** [comment]
Heya! That's working as expected. The status remains the same to make sure we don't leak any information on the existence of the items. There's a couple other discussions floating around around the ability to configure whether or not to expose that information, to make the API output a little "friendlier"
